public/v4/apps/litestream.yml

```diff
@@ -0,0 +1,70 @@
+captainVersion: 4
+services:
+    $$cap_appname:
+        caproverExtra:
+            dockerfileLines:
+                - FROM litestream/litestream:$$cap_version
+                - RUN mkdir /config
+                - |-
+                    RUN test -f /config/litestream.yml || echo -e "dbs:\n  - path: $$cap_db_path\n    replicas:\n      - type: s3\n        bucket: $$cap_bucket\n        path: $$cap_dest_path\n        endpoint: $$cap_endpoint_url\n        access-key-id: $$cap_access_key_id\n        secret-access-key: $$cap_secret_access_key" > /config/litestream.yml
+                - ENTRYPOINT ["/usr/local/bin/litestream"]
+                - CMD ["replicate", "-config", "/config/litestream.yml"]
+            notExposeAsWebApp: 'true'
+        volumes:
+            - $$cap_appname-data:/data
+            - $$cap_appname-config:/config
+        environment:
+            LITESTREAM_ACCESS_KEY_ID: $$cap_access_key_id
+            LITESTREAM_SECRET_ACCESS_KEY: $$cap_secret_access_key
+caproverOneClickApp:
+    variables:
+        - id: '$$cap_version'
+          label: Litestream Version
+          defaultValue: 'sha-1dbd8615908b1d3cebbfc44eb40a35056e15ba4c'
```

Comment:
Why not using one of their stable versions? 0.3 for example

Reply:
Absolutly, I missed that. Changed it to 0.3.

Reply:
Hi, the 0.4 version is released.